modules/eventing/pages/eventing-api.adoc

```diff
@@ -866,6 +866,24 @@ curl -XGET http://Administrator:password@192.168.1.5:8096/getFailureStats?name=[
 curl -XGET 'http://anyAuthedUser:password@192.168.1.5:8096/getFailureStats?name=[function_name]&bucket=[fs_bucket]&scope=[fs_scope]'
 ----
 
+| GET
+| [.path]_/resetStatsCounters?name=[function_name]_
```

Comment:
Any suggestions? we can revisit this later.  How do other pages do it?

Reply:
Agreed that it's something we can revisit and discuss later. I'm probably going to go back to the team and ask how we're handling this.

Reply:
This will be on the docket for a "revisit later" as I need to decide how we're going to do this from a style perspective. Going to merge this in the meantime.